server/lib/payments.js

```diff
@@ -389,7 +389,7 @@ const validatePayment = payment => {
     throw new Error('Interval should be null, month or year.');
   }
 
-  if (!payment.amount) {
+  if (!payment.amount && !payment.platformTip) {
```

Comment:
Sounds like we don't need this anymore?

```suggestion
  if (!payment.amount) {
```